lib/collection/src/shard.rs

```diff
@@ -567,15 +440,14 @@ impl Shard {
         self.segments.read().flush_all().unwrap();
         bar.finish();
     }
-}
 
-impl Drop for Shard {
```

Comment:
Is it possible to add a check into actual collection Drop that the `before_drop` was executed? So it should panic in the development mode.

Reply:
Found it, thanks!